mappings/net/minecraft/world/PortalForcer.mapping

```diff
@@ -1,13 +1,32 @@
 CLASS net/minecraft/unmapped/C_daetwwvm net/minecraft/world/PortalForcer
+	FIELD f_bfoygtnv NETHER_PORTAL_SPAWN_RADIUS I
+	FIELD f_emmqbhvf OVERWORLD_PORTAL_SPAWN_RADIUS I
```

Comment:
I feel like this may make more sense to be `PORTAL_SPAWN_RADIUS_OVERWORLD`, since saying `OVERWORLD_PORTAL...` might be confusing in some cases

Reply:
ig so sure